services/scabbard/libscabbard/src/service/factory.rs

```diff
@@ -47,7 +47,6 @@ pub struct ScabbardFactoryBuilder {
     signature_verifier_factory: Option<Arc<Mutex<Box<dyn VerifierFactory>>>>,
 }
 
-#[cfg(feature = "factory-builder")]
```

Comment:
Doesn't this have to match the cfg on the struct itself?

Reply:
Looks like I missed that one